python/sqlalchemy/src/yuga_demo.py

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker, relationship
+from http.server import HTTPServer, BaseHTTPRequestHandler
+import json
+from io import BytesIO
+import jsonpickle
+from yuga_demo_model import User, Order, Product, OrderLine
+
+## ######################################################
+## Configure the parameters below to connect to Yugabyte database: 
+## 
+listen_port = 8000
+db_user = 'postgres'
+db_password = 'pwd'
+database = 'alchemy_orm_sample'
```

Comment:
Lets make the default database name `yugabyte_alchemy`

Reply:
Done